spec/views/events/new.html.erb_spec.rb

```diff
@@ -1,67 +1,51 @@
 require 'rails_helper'
 
-RSpec.describe "events/new", type: :view do
-  context "new league" do
+RSpec.describe 'events/new', type: :view do
```

Comment:
Now that events/new routes to the create_from_type page we could add a test for that here. And also maybe add that for routing tests.

Reply:
This is a view spec, but yes, i should test the controller and routing